lib/msal-node/src/utils/Constants.ts

```diff
@@ -13,7 +13,18 @@ export enum HttpMethod {
 
 export enum HttpStatus {
     OK = 200,
-    REDIRECT = 302
+    OK_UPPER_LIMIT = 299,
+    REDIRECT = 302,
+    CLIENT_ERROR_LOWER_LIMIT = 400,
```

Comment:
Couple comments:
* CLIENT_ERROR_RANGE_START and CLIENT_ERROR_RANGE_END? Same pattern for others. 
* SUCCESS instead of OK?
* Why are we addressing only `302`? Do we not need to handle any other REDIRECTxx?

Reply:
The 302 is only being used in LoopbackClient, to "Prevent auth code from being saved in the browser history". I'll implement your syntax suggestions.